.env.local.example

```
@@ -1,20 +1,19 @@
-SEED_ADMIN_NAME=Administrator
+# Default User
+DEFAULT_USER_PASSWORD=P@ssw0rd
 
 # Guest Login
 GUEST_USER_EMAIL=guest@example.com
 GUEST_USER_PASSWORD=P@ssw0rd
+
+# Seed
+SEED_ADMIN_EMAIL=admin@example.com
+SEED_ADMIN_PASSWORD=P@ssw0rd
+SEED_ADMIN_NAME=Administrator
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SEED_ADMIN_NAME key should go before the SEED_ADMIN_PASSWORD key

(UnorderedKey)